Terran-One/cw-simulate-ui
# Fix saving chain config

## Issue link

[WL-456](https://terran-one.atlassian.net/browse/WL-XXX)

## Description

Updating chain config deletes the chain in the case where the chainId is unchanged. Added a check.

## Test steps

1. Update chain config without changing the ID, verify the chain is still there
2. Update chain config and change the ID, verify the old chain is replaced by a new one
